src/test/kotlin/string/addition/caculator/application/OperandParserTest.kt

```diff
@@ -0,0 +1,48 @@
+package string.addition.caculator.application
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.ValueSource
+import string.addition.caculator.domain.Operand
+
+class OperandParserTest {
+    @ParameterizedTest
+    @ValueSource(strings = ["1,2,3", "1:2:3", "1,2:3"])
+    fun `parse string by default delimiter`(inputStr: String) {
+        // given
+        val expected = (1..3).toList().map { Operand(it.toString()) }
```

Comment:
```suggestion
        val expected = (1..3).map { Operand(it.toString()) }
```
`..` 의 결과는 `IntRange`입니다. (`..`에 대고 cmd + B를 눌러보시면 확인이 가능합니다)
이는 `Iterable`의 구현체이기 때문에 바로 `Iterable#map`을 사용할 수 있어요!